polkadot-parachains/statemint/src/lib.rs

```diff
@@ -449,14 +449,17 @@ parameter_types! {
 	pub AssetsPalletLocation: MultiLocation =
 		PalletInstance(<Assets as PalletInfoAccess>::index() as u8).into();
 	pub CheckingAccount: AccountId = PolkadotXcm::check_account();
+	pub ParentAccount: AccountId = b"Parent"
+		.using_encoded(|b| AccountId::decode(&mut TrailingZeroInput(b)))
+		.expect("infinite length input; no invalid inputs for type; qed");
```

Comment:
Wonder if we can put this in cumulus or polkadot parachain primitives, make it a function and call it here instead.

Reply:
Why not just having `ParentIs` having doing this internally? (The type should then be renamed) But if someone needs a special parent account they can just create their own type that implements `Convert`

Reply:
Yeah, makes more sense that way.